src/electionguard/byte_padding.py

```diff
@@ -0,0 +1,54 @@
+from enum import IntEnum
+
+
+_PAD_BYTE = b"\x00"
+_BYTE_ORDER = "big"
+_PAD_INDICATOR_SIZE = 2
+
+
+class DataSize(IntEnum):
+    """Define the sizes for data."""
+
+    Bytes_512 = 512
```

Comment:
There is now a single data size used. The padding methods are all together and everything defaults to the only current size we support.